api/eds.proto

```diff
@@ -107,6 +99,17 @@ message LocalityLbEndpoints {
   // If unspecified, each locality is presumed to have equal weight in a
   // cluster.
```

Comment:
Unrelated, but these are actually weird semantics; what if some clusters have weights and others don't? Do they get an equal share of some residual pool? Since you're working in this area, can you clarify the comment?